bootstrap/bootstrap-storage.sh

```diff
@@ -40,34 +41,60 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-SCRIPT_RUNNER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.SQLScriptRunner
+STREAMLINE_TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.StreamlineTableInitializer
 for file in "${BOOTSTRAP_DIR}"/lib/*.jar;
 do
     CLASSPATH="$CLASSPATH":"$file"
 done
 
 echo "Using Configuration file: ${CONFIG_FILE_PATH}"
 
-function streamlineBootstrapStorage {
-    SCRIPT_DIR="${BOOTSTRAP_DIR}/sql/<dbtype>"
-    FILE_OPT="-f ${SCRIPT_DIR}/drop_tables.sql -f ${SCRIPT_DIR}/create_tables.sql"
+function dropStreamlineTables {
+    ${JAVA} -Dstreamline.bootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${STREAMLINE_TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} --drop-script ${SCRIPT_DIR}/drop_tables.sql --drop
+}
+
+function createStreamlineTables {
+    ${JAVA} -Dstreamline.bootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${STREAMLINE_TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} --create-script ${SCRIPT_DIR}/create_tables.sql --create
```

Comment:
It is not that easy to make the change from here. The script doesn't know which storage type the configuration sets up. Previously the script knows about storage type (via executing another Java class), but we abstracted it to the Java class to reduce two operations into one.

It shouldn't hurt to PostgreSQL storage setup, too.